mara_description/urdf/reinforcement_learning/mara_robot_gripper_140_camera_train.urdf

```diff
@@ -852,22 +852,15 @@
         </limit>
       </axis>
     </joint>
-    <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="hrim_actuation_gripper_000000000004">
-      <kp>500</kp>
+    <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="hrim_actuator_gripper_000000000004">
+      <kp>600</kp>
       <ki>0.1</ki>
       <kd>0.01</kd>
       <joint>robotiq_arg2f_base_to_left_inner_knuckle</joint>
       <joint>robotiq_arg2f_base_to_left_outer_knuckle</joint>
       <joint>robotiq_arg2f_base_to_right_inner_knuckle</joint>
       <joint>joint_finger</joint>
     </plugin>
-    <plugin filename="libgazebo_ros_joint_state_publisher.so" name="mara_joint_state">
```

Comment:
Why are you removing this plugin?

Reply:
It is using a different plugin, [mara_gazebo_train.urdf.xacro](https://github.com/AcutronicRobotics/MARA/blob/master/mara_description/urdf/reinforcement_learning/mara_gazebo_train.urdf.xacro).
That [plugin](https://github.com/AcutronicRobotics/MARA/blob/master/mara_description/urdf/reinforcement_learning/mara_gazebo_run.urdf.xacro) is only used in `run` models.

Reply:
In the RL for training we are using different approach where we do not need this plugin.